.travis.yml

```diff
@@ -15,7 +15,7 @@ services:
 env:
   global:
     - GO111MODULE=on
-    - GOLANGCI_LINT_VERSION=v1.31.0
+    - GOLANGCI_LINT_VERSION=v1.43.0
```

Comment:
Bumped golangci version to fix travis builds (`1.31` is missing `revive` plugin).